packages/@aws-cdk/core/lib/stack.ts

```diff
@@ -20,6 +20,8 @@ import { LogicalIDs } from './private/logical-id';
 import { resolve } from './private/resolve';
```

Comment:
The changes in this stack are also missing unit tests. Please add these as well.